DuckDuckGo/NetworkProtectionAccessController.swift

```diff
@@ -91,7 +76,7 @@ struct NetworkProtectionAccessController: NetworkProtectionAccess {
         }
 
         // Check for users who have activated the VPN via an invite code:
-        if networkProtectionActivation.isFeatureActivated && !networkProtectionWaitlistStorage.isInvited {
+        if networkProtectionActivation.isFeatureActivated {
             return .inviteCodeInvited
```

Comment:
Do we still need this? cc @samsymons

Reply:
Nah, this can be removed.

Reply:
@samsymons - Should I remove the entire file?  I was about to remove it but decided to ask in case there's a plan to add PPro support here.

Reply:
If we're managing access elsewhere this can be removed, sure. Access control is super simple now thanks to PPro.